data/json/items/ranged.json

```diff
@@ -3008,9 +3007,9 @@
     "recoil": 90,
     "durability": 7,
     "burst": 30,
-    "reload": 750,
+    "reload": 400,
     "loudness" : 160,
-    "magazines" : [ "stanag30", "belt223" ],
+    "magazines" : [ "stanag50", "stanag30", "belt223" ],
```

Comment:
Assuming this is one of the older variants of m249 that can use STANAG magazines. It should be able to use all STANAG compatible magazines.